src/Xamarin.Android.Build.Tasks/Tasks/GenerateJniRemappingNativeCode.cs

```diff
@@ -68,7 +68,7 @@ void Generate ()
 				XmlResolver = null,
 			};
 
-			using (var reader = XmlReader.Create (File.OpenRead (RemappingXmlFilePath.ItemSpec), readerSettings)) {
+			using (var reader = XmlReader.Create (File.OpenRead (RemappingXmlFilePath!.ItemSpec), readerSettings)) {
```

Comment:
Reading where we check if `RemappingXmlFilePath` is null:
https://github.com/dotnet/android/blob/27572cccbc5cd985a27900cd6722ef8b72695380/src/Xamarin.Android.Build.Tasks/Tasks/GenerateJniRemappingNativeCode.cs#L44-L52

Instead of using `!` here, can we pass in a non-nullable `string remappingXmlFilePath` to the `Generate()` method after the null check. Then we'd never have to check null later on?

Would that work?